src/lib/pages/proposal-details/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useRouter } from "next/router";
+
+import PageContainer from "lib/components/PageContainer";
+import { InvalidState } from "lib/components/state";
+
+import { ProposalDetailBody } from "./components/ProposalDetailsBody";
```

Comment:
add `index.ts` in the component dir with
```
export * from "./ProposalDetailsBody";
export * from "./ProposalTop";
```
then import from just `./components` instead